### DISCUSSION TITLE
Import spots detected with Deepblink in TrackMate

### DISCUSSION THREAD

**Discussion by irelamb** [BBQuercus/deepBlink#147] [Q&A] [answered]
Hi Bastian,
Thank you for the great tool you developed.
My question is the following: Deepblink deals with the tracking of diffraction limited spots. Can we directly import the resulting csv file in TrackMate for tracking the spots?

Thank you for your answer!

Irene

**Comment by BBQuercus** [comment, marked as answer]
Hi Irene,

Currently we don't have a nice way of converting things back to TrackMate since we typically use trackpy for tracking. Zhan wrote a script that does it however which was published in a recent publication: https://github.com/zhanyinx/SPT_analysis/blob/main/source/spot_detection_tracking/trackmate_xml_2d.py. Let me know if this is already useful and if yes we can add it to deepblink or at least mention it in more detail.

**Reply by irelamb** [reply]
Hi Bastian,
thank you very much for your quick reply.
I didn't know TrackPy. Until now I used TrackMate only, for both detection and tracking. It works very well for my images, but since I'd like to automatise my pipeline it may be interesting for me to try deepBlink in combination with TrackPy.
Thank you for Zhan script, I'll let you know if I try it.
